eng/common/scripts/artifact-metadata-parsing.ps1

```diff
@@ -157,220 +93,6 @@ function ResolvePkgJson($workFolder) {
   return ($pathsWithComplexity | Sort-Object -Property Complexity)[0].Path
 }
 
-# Parse out package publishing information given a .tgz npm artifact
```

Comment:
All the deleting scares me a little and so I think it would be best to do these changes incrementally. I think we can add a lot of the new functionality in parallel and once we can verify it works as specified then we can come in ad clean-up the unused stuff. 

My biggest concern is that I want to be sure that we don't break anything as we make these changes and I think the easiest way to do that is to do these changes incrementally.

Reply:
ok I see. That makes sense especially for servicing branches maybe. However I plan to test each language with a full template release before merging.